roles/create_rds_global_cluster/tasks/delete.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Run 'create_rds_global_cluster' role delete operations
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block:
+    - name: Get replica cluster info
+      amazon.aws.rds_cluster_info:
+        cluster_id: "{{ create_rds_global_cluster_replica_cluster_name }}"
+        region: "{{ create_rds_global_cluster_replica_cluster_region }}"
+      register: _result_replica_cluster_info
+
+    - name: If replica cluster exists
+      block:
+        - name: Delete instance attached to replica cluster if present
+          amazon.aws.rds_instance:
+            db_instance_identifier: "{{ create_rds_global_cluster_replica_cluster_instance_name }}"
+            region: "{{ create_rds_global_cluster_replica_cluster_region }}"
+            skip_final_snapshot: true
+            wait: false
+            state: absent
+          when: _result_replica_cluster_info.clusters[0].db_cluster_members | length >= 1
+
+        - name: Delete replica cluster without creating a final snapshot
+          amazon.aws.rds_cluster:
+            cluster_id: "{{ create_rds_global_cluster_replica_cluster_name }}"
+            region: "{{ create_rds_global_cluster_replica_cluster_region }}"
+            global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"
+            remove_from_global_db: true
+            skip_final_snapshot: True
+            state: absent
+      when: _result_replica_cluster_info.clusters | length != 0
+
+    - name: Get primary cluster info
+      amazon.aws.rds_cluster_info:
+        cluster_id: "{{ create_rds_global_cluster_primary_cluster_name }}"
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+      register: _result_primary_cluster_info
+
+    - name: If primary cluster exists
+      block:
+        - name: Delete instance attached to primary cluster if present
+          amazon.aws.rds_instance:
+            db_instance_identifier: "{{ create_rds_global_cluster_primary_cluster_instance_name }}"
+            region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+            skip_final_snapshot: true
+            wait: false
+            state: absent
+          when: _result_primary_cluster_info.clusters[0].db_cluster_members | length >= 1
```

Comment:
Same here. Why do we need this check here?

Reply:
I had added these so as.to have `extra checks` in place. but not required, removed